File: data/code_pairs/fortran/1715.f90
module ModTime
    use, intrinsic :: iso_fortran_env, only: real64
    implicit none

    real(real64) :: tSimulation = 0.0_real64
    integer :: iTimeArray(7) = 0
    real(real64) :: CurrentTime, EndTime, StartTime, VernalTime
    real(real64) :: PauseTime
    real(real64) :: utime
    integer :: iJulianDay, iDay
    integer :: iStep = 1

    integer, parameter :: iYear_   = 1
    integer, parameter :: iMonth_  = 2
    integer, parameter :: iDay_    = 3
    integer, parameter :: iHour_   = 4
    integer, parameter :: iMinute_ = 5
    integer, parameter :: iSecond_ = 6

end module ModTime

File: data/code_pairs/cpp/1715.cpp
#ifndef MODTIME_HPP
#define MODTIME_HPP

#include <array>

namespace ModTime {
    double tSimulation = 0.0;
    std::array<int, 7> iTimeArray{}; // Initialized with zeros

    double CurrentTime, EndTime, StartTime, VernalTime;
    double PauseTime;
    double utime; // Assuming it should be initialized to zero (not guaranteed in C++ without explicit initialization)
    int iJulianDay, iDay; // Assuming these should be initialized to zero for the test
    int iStep = 1;

    constexpr int iYear_   = 1;
    constexpr int iMonth_  = 2;
    constexpr int iDay_    = 3;
    constexpr int iHour_   = 4;
    constexpr int iMinute_ = 5;
    constexpr int iSecond_ = 6;
}

#endif // MODTIME_HPP